walmart/walmart.py

```diff
@@ -361,75 +362,65 @@ def get_cancel_payload(self, lines):
             ), xml_declaration=True, encoding='utf-8'
         )
 
-    def update_shipment(self, order_id, package):
-        headers = {
-            'Content-Type': "application/xml"
-        }
-        url = self.url + '/%s/shipping' % order_id
-        return self.connection.send_request(
-            method='POST',
-            url=url,
-            body=self.get_shipment_payload(package),
-            request_headers=headers
-        )
-
-    def get_shipment_payload(self, package):
-        """Shipment Update
-
-        :param package: {
-            "tracking_number": "",
-            "tracking_url": "",
-            "carrier": "",
-            "carrier_service": '',
-            "ship_date_time": '',
-            "items": [{
-                "line_number": "",
-                "quantity": 2,
-                "status": "Shipped",
-                "uom": "Each",
+    def ship(self, order_id, lines):
```

Comment:
update shipment is better

this sounds like you are instructing walmart to ship

Reply:
create_shipment may be?